tests/integration/targets/aci_file_remote_path/tasks/main.yml

```diff
@@ -0,0 +1,136 @@
+  cisco.aci.aci_file_remote_path:
+    <<: *aci_info
+    name: ansible_file_path
+    state: absent
+
+# CREATE FILE REMOTE PATH
+- name: Create a File Remote Path (check mode)
+  cisco.aci.aci_file_remote_path: &aci_path
+    <<: *aci_info
+    name: ansible_file_path
+    description: Test File Path
+    remote_host: test.example.com
+    remote_protocol: scp
+    remote_user: test_user
+    auth_type: password
+    remote_path: /tmp
+  check_mode: yes
+  register: cm_create_path
+
+- name: Create a File Remote Path
+  cisco.aci.aci_file_remote_path:
+    <<: *aci_path
+  register: nm_create_path
+
+- name: Create a File Remote Path Again
+  cisco.aci.aci_file_remote_path:
+    <<: *aci_path
+  register: nm_create_path_again
+
+- name: Verify creation of File Remote Path
+  ansible.builtin.assert:
+    that:
+    - cm_create_path is changed
+    - nm_create_path is changed
+    - nm_create_path_again is not changed
+    - nm_create_path.current.0.fileRemotePath.attributes.name == "ansible_file_path"
+    - nm_create_path.current.0.fileRemotePath.attributes.descr == "Test File Path"
+    - nm_create_path.current.0.fileRemotePath.attributes.protocol == "scp"
+    - nm_create_path.current.0.fileRemotePath.attributes.authType == "usePassword"
+    - nm_create_path.current.0.fileRemotePath.attributes.userName == "test_user"
+    - nm_create_path.current.0.fileRemotePath.attributes.remotePath == "/tmp"
+    - nm_create_path_again.current.0.fileRemotePath.attributes.name == "ansible_file_path"
+    - nm_create_path_again.current.0.fileRemotePath.attributes.descr == "Test File Path"
+    - nm_create_path_again.current.0.fileRemotePath.attributes.protocol == "scp"
+    - nm_create_path_again.current.0.fileRemotePath.attributes.authType == "usePassword"
+    - nm_create_path_again.current.0.fileRemotePath.attributes.userName == "test_user"
+    - nm_create_path_again.current.0.fileRemotePath.attributes.remotePath == "/tmp"
+
+# UPDATE FILE REMOTE PATH
+- name: Update File Remote Path
+  cisco.aci.aci_file_remote_path:
+    <<: *aci_path
+    remote_protocol: sftp
+    remote_user: new_user
+    remote_password: new_pass
+    remote_path: /tmp/dummy
+  register: update_path
+
+- name: Verify update of File Remote Path
+  ansible.builtin.assert:
+    that:
+    - update_path is changed
+    - update_path.current.0.fileRemotePath.attributes.name == "ansible_file_path"
+    - update_path.current.0.fileRemotePath.attributes.descr == "Test File Path"
+    - update_path.current.0.fileRemotePath.attributes.protocol == "sftp"
+    - update_path.current.0.fileRemotePath.attributes.authType == "usePassword"
+    - update_path.current.0.fileRemotePath.attributes.userName == "new_user"
+    - update_path.current.0.fileRemotePath.attributes.remotePath == "/tmp/dummy"
+
+# QUERY FILE REMOTE PATH
+- name: Query a File Remote Path
+  cisco.aci.aci_file_remote_path:
+    <<: *aci_path
+    state: query
+  register: query_one
+
+- name: Query all File Remote Paths
```

Comment:
Should we create more than 1 and test that query all indeed return more?

Reply:
test added